spec/views/login_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
```

Comment:
- [ ] Hi, Most of your tests are working well. However, you still have 8 out of your 32 example tests that are not passing please kindly fix these tests to ensure that your code will work the way it is supposed to. 

![image](https://user-images.githubusercontent.com/18730106/162507391-e889e87a-3266-406a-9d0d-781832440cc1.png)